coolbox/cli.py

```diff
@@ -189,7 +214,6 @@ def _fetch_frame_src(pos: str, source: str) -> Tuple[str, str]:
 
         return frame_var, source
 
-    @property
```

Comment:
The source property needs to be changed to a function, otherwise, call `coolbox` alone in bash will get into this function and raise error as no frames been add.